apps/api/src/domain/models/shared/common-command-errors/AggregateIdNotAvailableError.ts

```diff
@@ -8,6 +8,6 @@ import { AggregateId } from '../../../types/AggregateId';
  */
 export default class AggregateIdNotAvailableError extends InternalError {
     constructor(invalidId: AggregateId) {
-        super(`The ID: ${invalidId} is not available for use`);
+        super(`The Aggregate ID: ${invalidId} is not available for use`);
```

Comment:
I don't know if we need to be this specific. If you're going to change this, I'd just have an `IdNotAvailableError` instead of rewording this one. In the event this is an exception that boils up to the user, they don't know what an aggregate (root) is.

If you are going to reword it, make it `The ID: ${invalidId} is not available for use by an aggregate root`.

Reply:
This was just a suggestion, happy to go with what you think is best.  The issue I ran into was that the two errors, aggregate id and uuid, have the identical message so you can't tell them apart when they throw.  I was troubleshooting some seeded commands in cypress and had to figure out which of the two errors I was getting.  

I like `The ID: ${invalidId} is not available for use by an aggregate root` so I'll put that in.